pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl

```diff
@@ -169,6 +174,7 @@ task GenerateOutputMap {
     File input_bam
     String unmapped_bam_suffix
     Int disk_size
+    Int memory = 4
```

Comment:
Same comment about memory.   I think the VMs are sized at 3.75, 7.5 and 15 G.  So probably drop this down to 3.